consensus/src/txn_notifier.rs

```diff
@@ -49,17 +49,17 @@ impl MempoolNotifier {
 impl TxnNotifier for MempoolNotifier {
     async fn notify_failed_txn(
         &self,
-        block: &Block,
+        txns: Vec<SignedTransaction>,
         compute_results: &StateComputeResult,
     ) -> Result<(), MempoolError> {
         let mut rejected_txns = vec![];
-        let txns: Vec<_> = match block.payload() {
-            Some(payload) => payload,
-            None => return Ok(()),
-        }
-        .clone()
-        .into_iter()
-        .collect();
+        // let txns: Vec<_> = match block.payload() {
+        //     Some(payload) => payload,
+        //     None => return Ok(()),
+        // }
+        // .clone()
+        // .into_iter()
+        // .collect();
```

Comment:
Your change makes sense. We can remove this.

Reply:
Done